etc/picongpu/juwels-jsc/gpus_picongpu.profile.example

```diff
@@ -23,52 +22,46 @@ jutil env activate -p $proj
 # General modules #############################################################
 #
 module purge
-module load GCC/7.3.0
-module load CUDA/9.2.88
-module load CMake/3.15.0
-module load MVAPICH2/2.3-GDR
-module load Python/3.6.6
+
+module load GCC/8.3.0
+module load CUDA/10.1.105
+module load CMake/3.14.0
+module load MVAPICH2/2.3.3-GDR
+module load Python/3.6.8
+
+module load Boost/1.69.0-Python-3.6.8
+module load HDF5/1.10.5
+# necessary for evaluations (NumPy, SciPy, Matplotlib, SymPy, Pandas, IPython)
+module load SciPy-Stack/2019a-Python-3.6.8
 
 # Other Software ##############################################################
 #
-module load zlib/.1.2.11
-module load libpng/.1.6.35
-export CMAKE_PREFIX_PATH=$EBROOTZLIB:$EBROOTLIBPNG:$CMAKE_PREFIX_PATH
-
-# This is required for Boost to have correct dynamic library dependencies
-module load ICU/61.1
-export LD_LIBRARY_PATH=$EBROOTICU/lib:$LD_LIBRARY_PATH
-
-PARTITION_LIB=$PROJECT/lib_gpus
-BOOST_ROOT=$PARTITION_LIB/boost
-export CMAKE_PREFIX_PATH=$BOOST_ROOT:$CMAKE_PREFIX_PATH
-export LD_LIBRARY_PATH=$BOOST_ROOT/lib:$LD_LIBRARY_PATH
-
-HDF5_ROOT=$PARTITION_LIB/hdf5
-export PATH=$HDF5_ROOT/bin:$PATH
-export CMAKE_PREFIX_PATH=$HDF5_ROOT:$CMAKE_PREFIX_PATH
-export LD_LIBRARY_PATH=$HDF5_ROOT/lib:$LD_LIBRARY_PATH
-
-LIBSPLASH_ROOT=$PARTITION_LIB/libSplash
-PNGWRITER_ROOT=$PARTITION_LIB/pngwriter
+# Manually installed libraries are stored in PARTITION_LIB
+PARTITION_LIB=$PROJECT/lib
+
+
+LIBSPLASH_VER=1.7.0
+PNGWRITER_VER=0.7.0
+BLOSC_VER=1.19.0
+ADIOS_VER=1.13.1
+
+LIBSPLASH_ROOT=$PARTITION_LIB/LIBSPLASH/$LIBSPLASH_VER
+PNGWRITER_ROOT=$PARTITION_LIB/PNGWRITER/$PNGWRITER_VER
 export CMAKE_PREFIX_PATH=$LIBSPLASH_ROOT:$PNGWRITER_ROOT:$CMAKE_PREFIX_PATH
 
-BLOSC_ROOT=$PARTITION_LIB/c-blosc
+BLOSC_ROOT=$PARTITION_LIB/BLOSC/$BLOSC_VER
 export CMAKE_PREFIX_PATH=$BLOSC_ROOT:$CMAKE_PREFIX_PATH
 export LD_LIBRARY_PATH=$BLOSC_ROOT/lib:$LD_LIBRARY_PATH
 
-ADIOS_ROOT=$PARTITION_LIB/adios
+ADIOS_ROOT=$PARTITION_LIB/ADIOS/$ADIOS_VER
 export PATH=$ADIOS_ROOT/bin:$PATH
 export CMAKE_PREFIX_PATH=$ADIOS_ROOT:$CMAKE_PREFIX_PATH
 
 
-export LD_LIBRARY_PATH=$EBROOTICU/lib:$LD_LIBRARY_PATH
-
 # Environment #################################################################
 #
-#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
 
-export PICSRC=$HOME/src/picongpu
+export PICSRC=$HOME/shared/src/picongpu #modified
```

Comment:
why this comment ?

Reply:
My bad. Failed to remove in clean-up.

Reply:
why do you change the path to home shared? Is this a symlink to a projectfolder?